yb-voyager/cmd/exportData.go

```diff
@@ -221,45 +222,63 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName) er
 	if err != nil {
 		return fmt.Errorf("failed to start debezium: %w", err)
 	}
-	var status *dbzm.ExportStatus
-	exportingSnapshot := true
-	for exportingSnapshot {
-		status, err = debezium.GetExportStatus()
+
+	if liveMigration {
+		// monitor the debezium process and mode = snapshot/streaming
+		snapshotComplete := false
+		for debezium.IsRunning() {
+			if !snapshotComplete {
+				snapshotComplete, err = verifyAndHandleDebeziumSnapshotComplete(*debezium)
+				if err != nil {
+					return err
```

Comment:
why are we not exiting in this case?
Also, since we are returning from `debeziumExportData()` function, should we also call `d.Stop()` to ensure before exiting from the main function.

Reply:
Fixed. Calling utils.ErrExit, which calls atexit.Exit, which will ensure that the stop handler is run 👍